### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by hagoodj98** [community/community#127730] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hey I hosted my html project and everything on it looks good, except my background images. They are too big.  If you pull the code, you can see the background images just fine in developer tools. Need help. I haven't used github in a long time and may not knwo how to share code. I would like you to see the page so you get the understanding what I am talking about

**Comment by RajatRTC** [comment]
issue might be related to the CSS properties you're using to style the background images. However, I'll provide some general tips on how to handle background images to ensure they display correctly and responsively.

`body {
  background-image: url('your-background-image.jpg');
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;
}

/* media query to handle smaller screens */
@media (max-width: 768px) {
  body {
    background-size: contain;
  }
}
`

**Reply by hagoodj98** [reply]
Thanks for responding. I tried this way and it does not look good. I used dev tools and keep the background size to cover and it looks good on smaller devices. I would like to share the code so you know what I am talking about

**Comment by Abbas-Razaa** [comment]
To help you effectively, I need to see the HTML and CSS code related to your background images. If you're comfortable sharing your GitHub repository, you can provide the link, and I'll take a look at your code. Here's how you can share your code from GitHub:

Go to your GitHub repository.
Copy the URL from the address bar.
Paste the URL here.
If you're unsure about how to share the link, you can also copy the relevant HTML and CSS code snippets directly into your message. Here's an example format to follow:


<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" type="text/css" href="styles.css">
</head>
<body>
    <div class="background-image"></div>
</body>
</html>

`/* Example CSS */
.background-image {
    background-image: url('path/to/your/image.jpg');
    background-size: cover; /* or contain */
    background-repeat: no-repeat;
    background-position: center;
    width: 100%;
    height: 100vh; /* or a specific height */
}`

**Reply by hagoodj98** [reply]
https://github.com/hagoodj98/Carbon_Project.git

**Reply by hagoodj98** [reply]
What you will see in dev tools is what I want  and that is what's not in git hub pages. Thanks
